src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  static FULL_HEALTH = 100;
+
+  constructor(name) {
+    this.name = name;
+
+    this.health = Animal.FULL_HEALTH;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  static HIDDEN = false;
```

Comment:
you don't need make `HIDDEN`  like a static field

Reply:
Hi Volodymyr! Deleted static.